### ISSUE TITLE
Need control-f to search in document only (not passages)

### ISSUE THREAD

**Issue by gvcormac** [trec-dd/trec-dd-assessing#20]
At the moment, if I try to search the rendered document, the search looks through all the rendered passages on the right.  This makes search-within-document unusable.

**Comment by isoboroff** [comment]
Does clicking in the document frame before starting to search work?

**Comment by gvcormac** [comment]
Not that I noticed.  Certainly clicking does not restrict to that frame.  I tried to check it out but unable to create new subtopic at the moment.  The button is dead.

**Comment by shawn67** [comment]
did you create a topic?

**Comment by gvcormac** [comment]
Yes, I created a topic.

![screen shot 2015-04-12 at 6 42 50 pm](https://cloud.githubusercontent.com/assets/11865857/7107931/e36b6130-e143-11e4-96cb-dde0ad63b962.png)

**Comment by shawn67** [comment]
![image](https://cloud.githubusercontent.com/assets/8893176/7107944/4825d10a-e144-11e4-892c-b37105f5eaaa.png)

I can do it at my side; never occurred during my testing. Can you use chrome? What browser are you using?

**Comment by shawn67** [comment]
I assume your just work flow was 1) creating a topic candy; 2) search candy; 3) clicking add subtopic not working.

I repeated this process in assr8( with topic name candy2) in Chrome, not encountering this problem.

**Comment by gvcormac** [comment]
The problem is that immediately after you create the first topic, it says "topic created" but does not display the pull-down for topics -- it is stuck in "create topic" mode.  If you switch domains and back, the pull-down appears, along with the ability to create subtopics.

**Comment by shawn67** [comment]
so it's repeatable at your side right? what browser are you using, does chrome haslve the same problem

**Comment by gvcormac** [comment]
My workflow was
- create topic
- type in topic name
- get "topic created" (or "name collision")
- stuck in text box, no "add subtopic"
- do search 
- still stuck in text box
- do browse
- still stuck in text box
- switch to other domain
- switch to original tomain
- text box replace by pulldown

**Comment by gvcormac** [comment]
Maybe only for first topic.  Don't see it now.

**Comment by gvcormac** [comment]
Confirm.  Happened for first topic on assr5

**Comment by shawn67** [comment]
yeah, try another browser and what browser are you using? I tried accounts on my side assr8, not encountering this

**Comment by gvcormac** [comment]
Firefox.  I don't have Chrome.

**Comment by gvcormac** [comment]
Does not seem to happen on Safari.

**Comment by shawn67** [comment]
should be a cross-browser support problem. Although we can have assessors use the same browser, and I think we should, because during my development I used chrome all the time. But I'll fix this one for Firefox, looking into it. Thanks.

**Comment by gvcormac** [comment]
Confirm that the control-f problem exists even if you click the frame first.
